synfig-studio/src/gui/canvasview.cpp

```diff
@@ -1417,6 +1418,7 @@ CanvasView::init_menus()
 		{"jump-next-keyframe", "animate_seek_next_keyframe_icon", N_("Seek to Next Keyframe"),      "", sigc::mem_fun(*canvas_interface(), &CanvasInterface::jump_to_next_keyframe) },
 		{"jump-prev-keyframe", "animate_seek_prev_keyframe_icon", N_("Seek to Previous Keyframe") , "", sigc::mem_fun(*canvas_interface(), &CanvasInterface::jump_to_prev_keyframe) },
 
+		{"plugin-manager", "", N_("Plugin Manager"), "", sigc::mem_fun0(plugin_manager_dialog, &Dialog_PluginManager::present)}
```

Comment:
1. I guess you can remove the `0` digit of `sigc::mem_fun0()`, leaving it as `sigc::memfun()` only.
2. I think it's better to name it more descriptive as "open-plugin-manager".

Reply:
It's not possible to remove 0, because then I get error, 
```
more than one instance of overloaded function "sigc::mem_fun" matches the argument list
```